pkg/parser/types.go

```diff
@@ -0,0 +1,16 @@
+package parser
+
+/*
+MapOfMaps represents the structure of the parsed INI data,
+where the outer map's keys are section names, and the inner maps store key-value pairs for each section.
+*/
+type MapOfMaps map[string]map[string]string
```

Comment:
I used `MapOfMaps` to represent the hierarchical structure of INI files, with an outer map for sections and an inner map for key-value pairs. I thought it would make accessing and managing the data easier. If you think there’s a better approach or if you have concerns, I’m open to suggestions.